spec/decorators/artist_decorator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'Artist Decorator' do
+  describe 'decoration associations' do
+    it 'decorates associated songs' do
+      create(:song)
+      artists = Artist.includes(:songs).decorate
```

Comment:
Are you using `artists` here?

Think you can simply do

```
song = create :song
```

and then use that in your expectation test.

Factory girl will automatically create an artist for the song.